test/core/error_tracker.test.ts

```diff
@@ -128,7 +126,7 @@ test('works', () => {
     expect(tracker.getUniqueErrorCount()).toBe(1);
 });
 
-test('no code is null code', () => {
+test('no code is null code', async () => {
```

Comment:
why? whats the point of making the callback async if you didn't change anything inside the method itself?

Reply:
I forgot to change this back after removing `tracker.addAsync` function